pkg/features/kube_features.go

```diff
@@ -788,6 +788,13 @@ const (
 	// Allow users to specify whether to take nodeAffinity/nodeTaint into consideration when
 	// calculating pod topology spread skew.
 	NodeInclusionPolicyInPodTopologySpread featuregate.Feature = "NodeInclusionPolicyInPodTopologySpread"
+
+	// owner: @danwinship
+	// kep: http://kep.k8s.io/3178
+	// alpha: v1.25
+	//
+	// Causes kubelet to no longer create legacy IPTables rules
+	IPTablesCleanup featuregate.Feature = "IPTablesCleanup"
```

Comment:
The intention was to have this sorted, but seems that `NodeInclusion...` broke it cc: @pohly 

https://github.com/kubernetes/kubernetes/pull/109435

Reply:
somehow I managed to notice that in `defaultKubernetesFeatureGates` but missed it in the `const`s...